srcpkgs/pnpm/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'pnpm'
+pkgname=pnpm
+version=5.8.0
+revision=1
+# We won't use wrksrc, but it's needed to silence "cannot access wrksrc directory" error
```

Comment:
This isn't accurate, wrksrc is used as cwd of do_install, even if archive isn't extracted, please remove comment.

Reply:
Removed.